mp/src/game/client/momentum/ui/HUD/hud_mapfinished.cpp

```diff
@@ -20,6 +21,9 @@
 
 using namespace vgui;
 
+static MAKE_TOGGLE_CONVAR(mom_mapfinished_movement_enable, "0", FCVAR_CLIENTDLL | FCVAR_CLIENTCMD_CAN_EXECUTE | FCVAR_ARCHIVE,
+                          "Toggles being able to move. 0 = OFF, 1 = ON\n");
```

Comment:
Can we get a better string for this? It's pretty ambiguous:
```suggestion
                          "Toggles being able to move after completing a run. 0 = OFF, 1 = ON\n");
```